modules/internal/OwnedObject.chpl

```diff
@@ -291,6 +291,7 @@ module OwnedObject {
       }
     }
 
+    @deprecated("calling `.borrow()` on a type is deprecated - please use a cast to `borrowed` instead")
```

Comment:
Would it make more sense to say "on an `owned` type" instead of just "on a type"? I can imagine someone interpreting this as meaning something more widely applicable, such as implying there exists a deprecated `shared.borrow` type method.